frontend/src/components/InterviewPage.js

```diff
@@ -93,9 +93,7 @@ const InterviewPage = () => {
             <div className="container mt-5">
                 <ReactMarkdown>{questionDescription}</ReactMarkdown>
             </div>
-            <div>
-                <CodeEditor socket={collabSocket} roomId={interviewId} />
-            </div>
+            <CodeEditor socket={collabSocket} roomId={interviewId} />
             <Chat socket={chatSocket} username={username} room={interviewId} />
```

Comment:
should standardize room variable as either room or roomId, collab socket using roomId variable but chat socket using room variable.

Reply:
Can check with the rest if need to change